sympy/solvers/solveset.py

```diff
@@ -871,3 +872,295 @@ def solveset(f, symbol=None):
             return solveset_real(f, symbol)
         else:
             return solveset_complex(f, symbol)
+
+
+###############################################################################
+################################ LINSOLVE #####################################
+###############################################################################
+
+
+def linear_eq_to_matrix(equations, *symbols):
+    r"""
+    Converts a given System of Equations into Matrix form.
+    Here `equations` must be a linear system of equations in
+    `symbols`. The order of symbols in input `symbols` will
+    determine the order of coefficients in the returned
+    Matrix.
+
+    The Matrix form corresponds to the augmented matrix form.
+    For example:
+
+    4.x + 2.y + 3.z  = 1
```

Comment:
For docstrings I try to promote non-latex heavy equations so that they are readable from the terminal. I don't think SymPy has consensus on this but I find this much more preferable. Looking up online docs is one extra step that slows you down when coding and sometimes (if no Internet) you may not even have those available.